meson.options

```diff
@@ -8,6 +8,13 @@ option(
 )
 
 option('yamldir', type: 'string', description: 'Path to YAML')
+option(
+    'auditlog',
+    type: 'feature',
+    value: 'disabled',
```

Comment:
Ideally, you could leave this enabled by default so it would build as part of CI, but the auditd stuff would have to work there, and also on standalone builds that use the .wrap files int the subprojects dirs.

Reply:
Makes sense. I still have the wrap file change on my upstream todo list. I'll add the default enabling with the wrap change.